src/arrayMethodReduce.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let result;
+
+    if (initialValue >= 0) {
```

Comment:
```suggestion
    if (initialValue) {
```
тут может быть не только число, но и строка, и объeкт

Reply:
@vkryvytskyy 
если так сделать, то последний тест не проходит, вместо  4 получаем "а111''